### DISCUSSION TITLE
Error when trying to connect prisma with remote mysql hosted in cpanel

### DISCUSSION THREAD

**Discussion by MohamedFadelMoumeni** [prisma/prisma#16469] [Q&A]
Hello guys I have deployed a mysql database to shared server (Cpanel) and I'ved created a user for my db and grant all the permissions to it .
<img width="960" alt="2022-11-27_015727" src="https://user-images.githubusercontent.com/56879358/204115842-eda5f514-09c7-4688-a70a-552d4a5eda25.png">

and this is my db connection string : 

DATABASE_URL="mysql://USER_USERNAME:PASSWORD@IP_OF_SERVER:3306/db_name"

But I'm getting this error : 

P1001
Can't reach database server at `SERVER_IP`:`3306`
Please make sure your database server is running at `IP`:`3306`.

**Comment by mahbubhswe** [comment]
add your local mechin ip address to remote mysql connection.if not work contact with your hosting provider.......
![Screenshot_5](https://user-images.githubusercontent.com/99654070/204144920-4c4326f0-feea-4232-830f-3f86b933fdc8.jpg)

**Reply by pantharshit00** [reply]
👋 Hello @MohamedFadelMoumeni,  did you perhaps have a chance to check @mahbubhswe's recommendation? Let us know if you still have any questions!
 
Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
